packages/google_adsense/example/integration_test/test_ad.html

```diff
@@ -0,0 +1,4477 @@
+<!-- Copyright 2013 The Flutter Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file. -->
+
+<!DOCTYPE html>
```

Comment:
Option 1:
Mock Adsense.push() method that would
1. Inject custom JS on the setup or Use data-url instead of real AdSense SDK to load "fake" implementation
- with 100ms network simulation:
2. Return/render something smaller (div filled with color/image?)
3. Response as unfilled

Reply:
Assert the size of the platform view (0 or size of the ad)